### ISSUE TITLE
InternalError (see above for traceback): Blas SGEMM launch failed : m=43264, n=32, k=64

### ISSUE THREAD

**Issue by Rekito666** [Qidian213/deep_sort_yolov3#120]
InternalError (see above for traceback): Blas SGEMM launch failed : m=43264, n=32, k=64
         [[Node: conv2d_3/convolution = Conv2D[T=DT_FLOAT, _class=["loc:@batch_normalization_3/cond/FusedBatchNorm/Switch"], data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](leaky_re_lu_2/LeakyRelu, conv2d_3/kernel/read)]]
         [[Node: boolean_mask_39/GatherV2/_3383 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_6079_boolean_mask_39/GatherV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

Is there someone can help me?
win10 
tensorflow-gpu 1.9
cuda 9.0
cudnn 7.3.1

**Comment by Rekito666** [comment]
and RTX 2080 Ti

**Comment by FourierHe** [comment]
me too

**Comment by Theriyadh** [comment]
+3

**Comment by 2042644107** [comment]
Did you solve this problem?and how?please help me? @Rekito666 @FourierHe @Theriyadh
